config/config.go

```diff
@@ -77,6 +77,20 @@ const (
 	EnvAuthStrategy = "AUTH_STRATEGY"
 
 	EnvNamespaceLabelSelector = "NAMESPACE_LABEL_SELECTOR"
+
+	EnvLdapHost               = "LDAP_HOST"
+	EnvLdapPort               = "LDAP_PORT"
+	EnvLdapUseSSL             = "LDAP_USE_SSL"
+	EnvLdapInsecureSkipVerify = "LDAP_INSECURE_SKIP_VERIFY"
+	EnvLdapUserFilter         = "LDAP_USER_FILTER"
+	EnvLdapGroupFilter        = "LDAP_GROUP_FILTER"
+	EnvLdapBase               = "LDAP_BASE"
+	EnvLdapBindDN             = "LDAP_BIND_DN"
+	EnvLdapRoleFilter         = "LDAPROLEFILTER"
```

Comment:
This seems odd that all the rest of the env vars use "_" but this one "LDAPROLEFILTER" does not - wondering why this isn't also "LDAP_ROLE_FILTER".

Reply:
Changed this..